tests/test_read_alignments.py

```diff
@@ -1646,71 +1646,71 @@ def test_detect_with_library_format_unpaired(self):
         with self.assertRaisesRegex(ValueError, 'single end read'):
             self.make_ralign(library_format='ISF')
 
-    def test_detect_with_library_format_stranded_forward(self):
```

Comment:
why?

Reply:
Sorry, I should have done this in another branch. This was just me trying to debug failure due to illegal instruction.